Python/jit.c

```diff
@@ -56,9 +56,16 @@ jit_alloc(size_t size)
     int flags = MEM_COMMIT | MEM_RESERVE;
     unsigned char *memory = VirtualAlloc(NULL, size, flags, PAGE_READWRITE);
     int failed = memory == NULL;
+#elif defined(__APPLE__) && defined(__aarch64__)
```

Comment:
Does this work? If so, it seems a bit less fragile.

```suggestion
#elif defined(MAP_JIT)
```

I'd also like to reduce the duplication of code here. So maybe we can just *add* the flags in this case and use the same code path as before:

```c
#ifdef MAP_JIT
    flags |= MAP_JIT;
    prot |= PROT_EXEC;
#endif
```